.npmignore

```diff
@@ -0,0 +1 @@
+src/
```

Comment:
Ok. I'll change `["./lib"]` to `["lib"]` and `"./lib/index.js"` to `"lib/index.js"`.

I think that the untranspiled version should not be published to npm, at least not until the vast majority of javascript engines support ES6. That probably means waiting for a couple of years.